xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -502,6 +502,8 @@ void CGUIWindowPictures::GetContextButtons(int itemNumber, CContextButtons &butt
   CGUIMediaWindow::GetContextButtons(itemNumber, buttons);
   if (item && !item->GetProperty("pluginreplacecontextitems").asBoolean())
     buttons.Add(CONTEXT_BUTTON_SETTINGS, 5);                  // Settings
+
+  BaseContextMenuManager::Get().AppendVisibleContextItems(item, buttons);
```

Comment:
Yeah, it *can* be moved to `CGUIMediaWindow`, but then the addons will be added in the middle before the window specific ones instead of consistently last. I'm not sure what would be best..